tests/unit/Administration/Framework/Twig/ViteFileAccessorDecoratorTest.php

```diff
@@ -29,47 +28,41 @@ class ViteFileAccessorDecoratorTest extends TestCase
         ],
     ];
 
-    private MockObject $packagesMock;
+    private MockObject&\Symfony\Component\Asset\Package $packageMock;
 
     private ViteFileAccessorDecorator $decorator;
 
     protected function setUp(): void
     {
-        $this->filesystemMock = $this->createMock(FilesystemOperator::class);
-        $this->packagesMock = $this->createMock(Packages::class);
-        $this->packagesMock->method('getUrl')
+        $kernel = new StubKernel([
+            new BundleFixture('Administration', __DIR__ . '/Fixtures/Administration'),
+            new BundleFixture('TestBundle', __DIR__ . '/Fixtures/TestBundle'),
+        ]);
+
+        $this->packageMock = $this->createMock(UrlPackage::class);
+        $this->packageMock->method('getUrl')
             ->willReturn('https:://shopware.com/bundles/administration/');
 
         $this->decorator = new ViteFileAccessorDecorator(
             $this->configs,
-            $this->filesystemMock,
-            $this->packagesMock,
+            $this->packageMock,
+            $kernel
         );
     }
 
     #[DataProvider('hasFileProvider')]
-    public function testHasFile(string $configName, string $fileType, string $filePath, bool $fileExists): void
+    public function testHasFile(string $configName, string $fileType, bool $fileExists): void
     {
-        $this->filesystemMock->expects(static::once())
-            ->method('has')
-            ->with($filePath)
-            ->willReturn($fileExists);
-
-        $result = $this->decorator->hasFile($configName, $fileType);
-        static::assertEquals($fileExists, $result);
+        static::assertEquals($fileExists, $this->decorator->hasFile($configName, $fileType));
```

Comment:
please use `assertSame` where possible instead, as it is type safe

Reply:
Old habits die hard 🙈